modules/browser_object_reader_view.py

```diff
@@ -7,3 +9,71 @@ class ReaderView(BasePage):
     """
 
     URL_TEMPLATE = ""
+
+    def open_reader_view_searchbar(self) -> BasePage:
+        """
+        Opens the reader view using the search bar
+        """
+        before_page_source = self.driver.page_source
+        with self.driver.context(self.driver.CONTEXT_CHROME):
+            self.get_element("reader-view-button").click()
+        self.wait.until(lambda _: self.driver.page_source != before_page_source)
+        self.wait_for_reader_view_open()
+        return self
+
+    def open_reader_view_keys(self) -> BasePage:
+        """
+        Opens the reader view using keys
+        """
+        before_page_source = self.driver.page_source
+        with self.driver.context(self.driver.CONTEXT_CHROME):
+            if self.sys_platform() == "Darwin":
+                self.perform_key_combo(Keys.COMMAND, Keys.ALT, "r")
+            elif self.sys_platform() == "Linux":
+                self.perform_key_combo(Keys.CONTROL, Keys.ALT, "r")
+            else:
+                self.perform_key_combo(Keys.F9)
+        self.wait.until(lambda _: self.driver.page_source != before_page_source)
+        self.wait_for_reader_view_open()
+        return self
+
+    def close_reader_view_searchbar(self) -> BasePage:
+        """
+        Closes the reader view using the search bar
+        """
+        before_page_source = self.driver.page_source
+        with self.driver.context(self.driver.CONTEXT_CHROME):
+            self.get_element("reader-view-button").click()
+        self.wait.until(lambda _: self.driver.page_source != before_page_source)
+        self.wait_for_reader_view_closed()
+        return self
+
+    def close_reader_view_keys(self) -> BasePage:
+        """
+        Closes the reader view using keys
+        """
+        before_page_source = self.driver.page_source
+        with self.driver.context(self.driver.CONTEXT_CHROME):
+            if self.sys_platform() == "Darwin":
+                self.perform_key_combo(Keys.COMMAND, Keys.ALT, "r")
+            elif self.sys_platform() == "Linux":
+                self.perform_key_combo(Keys.CONTROL, Keys.ALT, "r")
+            else:
+                self.perform_key_combo(Keys.F9)
+        self.wait.until(lambda _: self.driver.page_source != before_page_source)
+        self.wait_for_reader_view_closed()
+        return self
+
+    def wait_for_reader_view_open(self) -> BasePage:
+        """
+        Checks to see if the reader view toolbar is present demonstrating that the reader view is open.
+        """
+        self.wait.until(lambda _: self.element_exists("reader-toolbar"))
```

Comment:
`element_exists` and `element_does_not_exist` use WebDriverWait objects under the hood, we don't need to nest them. That is, this line could just be `self.element_exists("reader-toolbar")`